ImageEditor/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,7 @@
     <application>
         <activity
             android:name=".EditImageActivity"
+            android:configChanges="orientation|screenSize"
```

Comment:
Can you please elaborate why we want to handle the configuration changes manually?

Reply:
Fixed here acd9e6a